src/site/paragraphs/q_a_section.drupal.liquid

```diff
@@ -12,7 +12,7 @@
   }
 {% endcomment %}
 {% if entity.fieldSectionHeader != empty %}
-  <h2 class="heading-level-3">{{ entity.fieldSectionHeader }}</h2>
+  <h2 {% if entity.fieldAccordionDisplay %}class="heading-level-3"{% endif %} >{{ entity.fieldSectionHeader }}</h2>
```

Comment:
Just curious, but are there cases where `fieldAccordionDisplay` isn't true?

Reply:
yes, there are. 
I believe `fieldAccordionDisplay` comes through as a boolean. @ahay-agile6 can you confirm?